Failure Modules/TestFlightFailure_SolarTracking.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+using KSP;
+
+namespace TestFlight.Failure_Modules
+{
+    public class TestFlightFailure_SolarTracking : TestFlightFailureBase_Solar
+    {
+        public override void DoFailure()
+        {
+            base.DoFailure();
+            base.module.trackingSpeed = 0;
+        }
+        public override float DoRepair()
+        {
+            base.DoRepair();
+            if (base.part != null && base.part.partInfo != null)
+            {
+                Part pf = base.part.partInfo.partPrefab;
+                if (pf != null)
+                {
+                    ModuleDeployableSolarPanel pm = (ModuleDeployableSolarPanel)pf.Modules.OfType<ModuleDeployableSolarPanel>();
+                    if (pm != null)
+                    {
+                        base.module.trackingSpeed = pm.trackingSpeed;
```

Comment:
any reasoning behind that thinking? before i used to save the value, but since the field trackingSpeed isnt persistent, and nothing to tweak with it, figured better to use the prefab. once failed state of failures is fixed to save, id have to save the cached value as well. 1) adding more data to the save 2) messing up any changes from configs that change the part. if speed is 5, fail saves to 0, new mod added changes to 10, repair would put back 5, until restart where 10 fills in anyways.

feel free to say i overthink it, just what felt right, i'm not attached to either method, so feel free to discuss :)